DBM-Core/DBM-Core.lua

```diff
@@ -41,7 +41,7 @@
 -------------------------------
 --  Globals/Default Options  --
 -------------------------------
-DBM_REVISION = "$Revision: 4449 $"
+DBM_REVISION = "$Revision: 4450 $"
 
```

Comment:
And well, this revision has to be brought back to 4449 or we will skip revision.